styles/banner.less

```diff
@@ -0,0 +1,124 @@
+@import "mixins.less";
+@shadow-width: 9px; 
+.banner__slider{
+    max-width: @width_viewScreen;
+    margin: auto;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    &_leftArrow{
+        .arrow();
```

Comment:
Maybe you could create an arrow as an element, and left/right arrows as modificators? You could even have 1 background image, and use `transform: rotate(180deg);`